.github/workflows/sync-release-assets.yml

```diff
@@ -10,7 +10,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  sync-github-and-dist.ipfs.tech:
+  sync-github-and-dist-ipfs-tech:
```

Comment:
@guseggert  i did not git blame but pretty sure it was me not escaping `.` late in the evening while renaming everyhing to .tech :crying_cat_face: 

![woke-up-but-at-what-cost](https://user-images.githubusercontent.com/157609/187297491-c471dc3b-b536-4f7b-aefc-0ca2ca5b1ce5.png)

Reply:
```
find . -type f -name "*.go" | parallel sed -i "s,dist.ipfs.io,dist.ipfs.tech,g" {}
```